feat: possibility to disable for certain medias

QuentinFringhian opened the pull request:
This pr adds the possibility to disable the resize-image quill plugin for certain formats. The configuration have been added to the option argument of the plugin.

The files from the dist folder were automatically generated with npm run build.

Thank you for your time and have a nice day.

hunghg255 commented:
I think dont need push dist folder

QuentinFringhian commented:
> I think dont need push dist folder

Got it ! i removed it from my branch !

hunghg255 commented:
> > I think dont need push dist folder
> 
> Got it ! i removed it from my branch !

still conflict

QuentinFringhian commented:
> > > I think dont need push dist folder
> > 
> > 
> > Got it ! i removed it from my branch !
> 
> still conflict

ah yes of course, my bad, it is now fixed ! thank you for your patience !
